articles/how-to-create-applications-for-azure-quantum.md

```diff
@@ -42,90 +36,80 @@ For example, this operation can NOT be run on a No Control Flow target:
     }
 ```
 
-Trying to run this operation on a No Control Flow target will fail because it evaluates a comparison between two results (`desired != M(q)`)
-to control the computation flow with an `if` statement.
+Trying to run this operation on a No Control Flow target will fail because it evaluates a comparison between two results (`desired != M(q)`) to control the computation flow with an `if` statement.
 
 > [!NOTE]
-> Currently, there is an additional limitation for this type of profile target: *you can't apply operations on qubits that have been measured, even
-> if you don't use the results to control the program flow.* This limitation is
-> not inherent to this profile type but is circumstantial to the situation of the Public
-> Preview.
+> Currently, there is an additional limitation for this type of profile target: *you can't apply operations on qubits that have been measured, even if you don't use the results to control the program flow.* This limitation is not inherent to this profile type but is circumstantial to the situation of the Public Preview.
 
 Presently, these No Control Flow targets are available for Azure Quantum:
 
 - **Provider:** IonQ
   - IonQ simulator (`ionq.simulator`)
   - IonQ QPU: (`ionq.qpu`)
 
-### Create applications for IonQ targets
+## Create and run applications for Basic Measurement Feedback targets
+
+**Basic Measurement Feedback** profile targets can run a wide variety of Q# applications, with the constraint that you can only compare values of type `Result` as part of conditions within `if` statements in operations. The corresponding conditional blocks may not contain `return` or `set` statements. This profile type supposes an improvement over No Control Flow profiles, but still is subject to some limitations.
```

Comment:
```suggestion
**Basic Measurement Feedback** profile targets can run a wide variety of Q# applications, with the constraint that you can only compare values of type `Result` as part of conditions within `if` statements in operations. The corresponding conditional blocks may not contain `return` or `set` statements. This profile type supposes an improvement over **No Control Flow** profiles, but still is subject to some limitations.
```

Reply:
We could bold other mentions of the profile names, but currently don't seem to be doing so (e.g. L44 and L39 are not bolded either). I guess only where we introduce it for the first time.

Reply:
Good then :)